source/Ship.cpp

```diff
@@ -275,29 +275,61 @@ void Ship::Load(const DataNode &node)
 				if(child.Size() >= 2)
 					outfit = GameData::Outfits().Get(child.Token(1));
 			}
-			Angle gunPortAngle = Angle(0.);
-			bool gunPortParallel = false;
+			Hardpoint::BaseAttributes attributes;
+			attributes.baseAngle = Angle(0.);
+			attributes.isParallel = false;
+			attributes.isOmnidirectional = true;
 			bool drawUnder = (key == "gun");
 			if(child.HasChildren())
 			{
+				bool defaultBaseAngle = true;
 				for(const DataNode &grand : child)
 				{
+					bool needToCheckAngles = false;
```

Comment:
Is this name "needToCheckAngles" a fit for the purpose?

Reply:
Maybe? Not too sure what would be a better and clearer name. It's quite close to where it was used anyway so I don't think there would be much confusion from it.

Reply:
Probably clear enough.